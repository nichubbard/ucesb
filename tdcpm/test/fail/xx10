/* ### Unit mismatch */

/* Fail due to wrong unit. */
r.a = 7 cm;

