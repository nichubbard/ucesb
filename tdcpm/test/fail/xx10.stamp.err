0x558b0daf82a0 7.0000
Unit mismatch, cannot assign.
