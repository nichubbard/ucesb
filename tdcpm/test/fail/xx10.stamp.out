r.a = 7 cm;
